content/blog/2022-2023/2022-06-05-ccss-summer-dev-projects-introduction.md

```diff
@@ -31,3 +31,14 @@ part of the Rust Gamedev Working Group, and the team lead of the Rust CTCFT
 team. I’m also one of the core developers on Veloren, an open source game
 similar to Minecraft. In the Rust communty, I run a monthly game dev meetup, and
 co-edit the monthly game dev newsletter.
+
+### Nguyen-Hanh Nong ([@Nguyen-HanhNong](https://github.com/Nguyen-HanhNong))
+
+Sup, my name is Nguyen-Hanh. :handshake:
+
+I'm a honours computer science student at Carleton University, specializing in A.I and Machine Learning.
```

Comment:
Nice, I'm also in the AI stream. Can't wait to be able to take courses with you!!